docs/files.md

```diff
@@ -55,6 +55,8 @@ The `graph.json` is an output file that contains all the paths fromager can take
 
 The format is a mapping from resolved versions expressed as a requirement specification (package==version) to a mapping with properties of the package and the list of dependencies of that package.
 
+The `graph.json` file is also used for repeatable builds feature that fromager supports. Fromager bootstrap can pull an already existing `graph.json` file from a previous bootstrap using a tag and the bootstrap command can use `--previous-bootstrap-file` option to generate a repeatable wheel build.
```

Comment:
```suggestion
The `graph.json` file is also used for the "repeatable builds" feature. Fromager bootstrap can pull an already existing `graph.json` file from a previous bootstrap via `--previous-bootstrap-file` option to generate a repeatable wheel build.
```

We happen to use tags and releases downstream, but that's just how we've set up our builds. That isn't actually fundamental to the way the feature works in fromager.

Reply:
Oh okay, got it